src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -56,6 +65,10 @@
     }
   }
 
+  .p-modal .p-list__link {
+    display: block;
```

Comment:
You should be able to remove this once you add `p` tags.

Reply:
yep removed.